doc/stdenv.xml

```diff
@@ -179,68 +179,180 @@ genericBuild
 </section>
 
 
-<section xml:id="ssec-stdenv-attributes"><title>Attributes</title>
-
-<variablelist>
-  <title>Variables affecting <literal>stdenv</literal>
-  initialisation</title>
-
-  <varlistentry>
-    <term><varname>NIX_DEBUG</varname></term>
-    <listitem><para>If set, <literal>stdenv</literal> will print some
-    debug information during the build.  In particular, the
-    <command>gcc</command> and <command>ld</command> wrapper scripts
-    will print out the complete command line passed to the wrapped
-    tools.</para></listitem>
-  </varlistentry>
+<section xml:id="ssec-stdenv-dependencies"><title>Specifying dependencies</title>
 
-</variablelist>
+<para>
+  As described in the Nix manual, almost any drv file store path in the a new derivation's attribute set will in fact induce a dependency on that derivation.
+  <varname>mkDerivation</varname>, however, takes a few attributes intended to, between them, include all the dependencies of a package.
+  This is done both for structure and consistency, but also so that certain other setup can take place.
+  For example, certain dependencies need their bin directories added to the <envar>PATH</envar>.
+  That is built-in, but other setup is done via a pluggable mechanism that works in conjunction with these dependency attributes.
+  See <xref linkend="ssec-setup-hooks"/> for details.
+</para>
+<para>
+  The dependencies can be broken down along two axes: the dependencies platforms relative to the new derivation's, and whether the dependencies are propagated.
```

Comment:
> the dependencies platforms relative to the new derivation's

At least one apostrophe is missing and possibly one is too much, I don't get that sentence..